buildSrc/src/main/groovy/org/apache/beam/gradle/BeamModulePlugin.groovy

```diff
@@ -383,7 +383,7 @@ class BeamModulePlugin implements Plugin<Project> {
     def google_cloud_spanner_version = "1.49.1"
     def google_http_clients_version = "1.34.0"
     def grpc_version = "1.25.0"
-    def guava_version = "25.1-jre"
+    def guava_version = "28.0-jre"
```

Comment:
Why to not use the latest `28.2-jre` version?

Reply:
This was because `28.0-jre` is the first version having missing symbols. `28.2-jre` would work too. It's up to Beam's team to decide which version to go.

Reply:
Either is fine, if all the tests and work were done to update to 28.0-jre have been done then it makes sense to go with 28.0-jre. But if there is a bunch of investigation that is necessary. Then its up to the PR author to choose whether they want to restart the tests/validation from 28.2-jre.

As a reviewer, I would be happy with either the 28.0-jre or 28.2-jre upgrade.